src/Amalgam/evaluablenode/EvaluableNode.h

```diff
@@ -19,6 +19,7 @@
 #include <string>
 #include <vector>
 
+#define AMALGAM_FAST_MEMORY_INTEGRITY
```

Comment:
Should this be here? If I'm reading the comment correctly, wouldn't this enable the fast memory integrity checks all the time?